06_gpu/stable_diffusion_cli.py

```diff
@@ -41,11 +67,10 @@
         ]
     )
     .run_commands(["pip install diffusers[torch] transformers ftfy accelerate"])
-    .run_commands(
-        [
-            "curl -L https://gist.github.com/luiscape/36a8cd29b8ed54cfbfcf56d51fe23cc0/raw/a6bf16996efe7c59114eea7944b0f99741d83d54/download_stable_diffusion_models.py | python"
-        ],
+    .run_function(
+        download_models,
         secrets=[modal.Secret.from_name("huggingface-secret")],
+        gpu=modal.gpu.A100(),
```

Comment:
I believe that this step doesn't need to run on a GPU.

Reply:
It doesn't have to, but there's a minor warning when you don't.

```
/usr/local/lib/python3.9/site-packages/diffusers/models/attention.py:433: UserWarning: Could not enable memory efficient attention. Make sure xformers is installed correctly and a GPU is available: torch.cuda.is_available() should be True but is False. xformers' memory efficient attention is only available for GPU
```